### DISCUSSION TITLE
Kali with some sort of XRDP ?

### DISCUSSION THREAD

**Discussion by clemdepernet** [gravitational/teleport#50007] [Q&A] [answered]
Hello !
Just started setuping my Teleport Bastion and already enrolled few ressources. The idea is to get a lab ready anytime.
I tried figuring out how to get Kali(GUI) in the same way Windows Remote Access works on teleport. (I've got a Kali Instance on AWS)

Thing is, I don't really know where to start, and I also never seen someone do it elsewhere.
Any ideas ?

**Comment by webvictim** [comment, marked as answer]
Assuming you mean the GUI for Kali Linux, Remote Desktop access for Linux isn't supported today: https://github.com/gravitational/teleport/issues/13179

The easiest workaround is probably to run VNC on the machine and expose it with TCP application access: https://goteleport.com/docs/enroll-resources/application-access/guides/tcp/

**Reply by clemdepernet** [reply]
Alright, thanks I'll take a look.
